.github/workflows/check-commits

```diff
@@ -0,0 +1,134 @@
+#!/bin/sh
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 Robin Jarry
+
```

Comment:
@karthiksundaravel / @dsneddon - can we use copyright statement like this ? We have these in other files - eg; -https://github.com/os-net-config/os-net-config/blob/master/os_net_config/sriov_bind_config.py#L1

Reply:
Perhaps @rjarry would be willing to make this available under the same license as the rest of os-net-config?

Reply:
This is the same license. Only a different copyright. I removed it. It makes no sense to add a license to such kind of script :)